### DISCUSSION TITLE
Transition problem Ikea ICPSHC24-10EU-IL-1

### DISCUSSION THREAD

**Discussion by Kevguy91** [Koenkk/zigbee2mqtt#17210] [Devices]
Hello everybody,

When I create a group scene including the ikea driver, there is a problem of duration during the transition of the scene (only on the ikea driver, not on the other bulbs, Hue).

I also have this problem if I create a scene for the driver alone.

However, I don't have this problem when manually on/off. The transition duration corresponds to the value entered in the "transition" field of the device setting.

Anyone encounter this bug?

**Comment by Kevguy91** [comment]
I revive the topic, does anyone have this problem?

**Comment by Wikibear** [comment]
I have some issues also with lights since update to 1.30.1 can you confirm maybe this:
https://github.com/Koenkk/zigbee2mqtt/issues/17405
I must set the brightness and something like this first and then everthing will work like expected until the next restart.
Maybe your groups expect the same issue.

**Comment by Kevguy91** [comment]
I don't seem to have this problem, all individual or group lights work fine after restarting z2m. The problem is only on the transition of a scene.
